nautobot/extras/forms/forms.py

```diff
@@ -605,10 +609,16 @@ class ExternalIntegrationBulkEditForm(NautobotBulkEditForm):
     verify_ssl = forms.NullBooleanField(required=False, label="Verify SSL", widget=BulkEditNullBooleanSelect)
     timeout = forms.IntegerField(required=False, min_value=0)
     extra_config = forms.JSONField(required=False)
+    http_method = forms.ChoiceField(
+        required=False,
+        label="HTTP Method",
+        choices=add_blank_choice(WebhookHttpMethodChoices),
+    )
+    headers = forms.JSONField(required=False, label="HTTP Request headers")
```

Comment:
Sorry, I should have responded more promptly. My reasoning is that since we are using a custom template for Edit/Create ExternalIntegration, the form field's order are hard-coded in the template anyway. So I thought the order here on the form did not matter. Please correct me if I am wrong.

Reply:
Makes sense. Thanks :-)